fms/models/gpt_bigcode.py

```diff
@@ -33,7 +33,7 @@ class GPTBigCodeConfig(ModelConfig):
     linear_config: Optional[
         Mapping[str, Any]
     ] = None  # pass as {"linear_type": str, <other kwargs>}
-    unfuse_strategy: Optional[str] = None  # TODO: could be an Enum
+    fused_weights: bool = True
```

Comment:
Do we expect weights to be stored unfused?

Reply:
with finetuning on Spyre, where we are training with unfused weights on FMS scripts, yes